### PR TITLE
chore: cleanup

### PR DESCRIPTION
**Pull request by CodeWithEmad**
This PR will introduce the following:

- Remove trailing whitespaces
- fix typos
- add codeblock for better readability
- fix heading styles

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @CodeWithEmad!

### What's next?

*Please work through the following steps to get your changes ready for engineering review:*

#### :radio_button: Get product approval

If you haven't already, [check this list](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Does-my-contribution-require-Product-Review%3F) to see if your contribution needs to go through the product review process.

- If it does, you'll need to submit a product proposal for your contribution, and have it reviewed by the [Product Working Group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3449028609/Product+Working+Group).
    - This process (including the steps you'll need to take) is documented [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Product-Review-Process).
- If it doesn't, simply proceed with the next step.

#### :radio_button: Provide context

To help your reviewers and other members of the community understand the purpose and larger context of your changes, feel free to add as much of the following information to the PR description as you can:

- Dependencies
  > This PR must be merged before / after / at the same time as ...
- Blockers
  > This PR is waiting for OEP-1234 to be accepted.
- Timeline information
  > This PR must be merged by XX date **because** ...
- Partner information
  > This is for a course on edx.org.
- Supporting documentation
- Relevant [Open edX discussion forum](https://discuss.openedx.org/) threads





#### :radio_button: Get a green build

If one or more checks are failing, continue working on your changes until this is no longer the case and your build turns green.



#### :radio_button: Let us know that your PR is ready for review:

### Who will review my changes?


This repository is currently maintained by `@sarina`.  Tag them in a comment and let them know that your changes are ready for review.


### Where can I find more information?

If you'd like to get more details on all aspects of the review process for open source pull requests (OSPRs), check out the following resources:

- [Overview of Review Process for Community Contributions](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/FAQ-about-pull-requests.html)
- [Pull Request Status Guide](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/pull-request-statuses.html)
- [Making changes to your pull request](https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html)

### When can I expect my changes to be merged?

Our goal is to get community contributions seen and reviewed as efficiently as possible.

However, the amount of time that it takes to review and merge a PR can vary significantly based on factors such as:

- The size and impact of the changes that it introduces
- The need for product review
- Maintenance status of the parent repository

:bulb: *As a result it may take up to several weeks or months to complete a review and merge your PR.*

**Comment by sarina** [comment]
@CodeWithEmad thank you! I will review later this week.

File: oeps/best-practices/oep-0066/Open_edX_Auth_Overview_Table.rst [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Open edX Auth Overview Table
```
Comment (sarina) [inline oeps/best-practices/oep-0066/Open_edX_Auth_Overview_Table.rst:1]: Please don't change/rename this file

File: oeps/processes/oep-0055-proc-project-maintainers.rst [hunk 2]
```diff
@@ -144,7 +144,7 @@ Historical Context
 Historically, the Open edX project has not had a well-defined role
 of maintainer.  At times, community members stepped into this role
 informally.  The role was also part of a broader "owner" role at edX.
-Owners were also responsible for the health of the component they
+Owners were also responsible for the health of the components they
```
Comment (sarina) [inline oeps/processes/oep-0055-proc-project-maintainers.rst:147]:
```suggestion
Owners were also responsible for the health of the component(s) they
```

File: oeps/processes/oep-0055-proc-project-maintainers.rst [hunk 3]
```diff
@@ -479,9 +479,9 @@ Notes
        can be seen in much literature that starts from the assumption
        that maintainers are essential and instead asks, what makes a
        maintainer great, what are the best-practices maintainers
-       should follow, how do we ensure maintainer don't burn out, how
+       should follow, how do we ensure maintainers don't burn out, how
        do we ensure critical parts of the software supply chain, like
-       OpenSSL are maintained, etc.  I have included some relevant
+       OpenSSL is maintained, etc.  I have included some relevant
```
Comment (sarina) [inline oeps/processes/oep-0055-proc-project-maintainers.rst:484]: Rervert this change ("are" is referring to the plural "critical parts")

**Review by sarina — CHANGES_REQUESTED** [review]

File: oeps/best-practices/oep-0066/Open_edX_Auth_Overview_Table.rst (hunk 1 above)
Comment (CodeWithEmad) [reply oeps/best-practices/oep-0066/Open_edX_Auth_Overview_Table.rst:1]: You got it!

File: oeps/processes/oep-0055-proc-project-maintainers.rst (hunk 3 above)
Comment (CodeWithEmad) [reply oeps/processes/oep-0055-proc-project-maintainers.rst:484]: You're absolutely right!

File: oeps/processes/oep-0055-proc-project-maintainers.rst (hunk 2 above)
Comment (CodeWithEmad) [reply oeps/processes/oep-0055-proc-project-maintainers.rst:147]: Sure

**Review by sarina — APPROVED** [review]
Thanks for this cleanup!

**Pull request merged**